### PR TITLE
Added 1.21-2 docs

### PR DESCRIPTION
**Pull request by kschumy**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by kschumy** [comment]
Holding until ready for this step

**Review by TerryHowe — APPROVED** [review]
/lgtm

File: docs/contents/index.md [hunk 1]
```diff
@@ -120,6 +120,7 @@ repository for the [EKS Optimized AMI](https://github.com/awslabs/amazon-eks-ami
 if you are interested in the AL2 container runtime kernel version.
 
 #### EKS-D 1.21 Version Dependencies
+* [v1-21-eks-2](releases/1-21/2/index.md) (July 26, 2021)
```
Comment (kschumy) [inline docs/contents/index.md:123]:
```suggestion
* [v1-21-eks-2](releases/1-21/2/index.md) (July 27, 2021)
```

**Pull request merged**